src/ViewHoldingsRecord.js

```diff
@@ -460,51 +469,87 @@ class ViewHoldingsRecord extends React.Component {
           'Holdings statement for supplements note': intl.formatMessage({ id: 'ui-inventory.holdingsStatementForSupplementsNote' }),
         },
         columnWidths: { 'Holdings statement for supplements': '16%' },
         formatter: {
-          'Holdings statement for supplements': x => get(x, ['statement'], ''),
-          'Holdings statement for supplements note': x => get(x, ['note'], ''),
+          'Holdings statement for supplements': x => get(x, ['statement']) || noValue,
+          'Holdings statement for supplements note': x => get(x, ['note']) || noValue,
         },
         ariaLabel: intl.formatMessage({ id: 'ui-inventory.holdingsStatementForSupplements' }),
       },
       {
         id: 'list-holdingsstatementsforindexes',
-        contentData: holdingsDetails.holdingsStatementsForIndexes,
+        contentData: checkIfArrayIsEmpty(holdingsDetails.holdingsStatementsForIndexes),
         visibleColumns: ['Holdings statement for indexes', 'Holdings statement for indexes note'],
         columnMapping: {
           'Holdings statement for indexes': intl.formatMessage({ id: 'ui-inventory.holdingsStatementForIndexes' }),
           'Holdings statement for indexes note': intl.formatMessage({ id: 'ui-inventory.holdingsStatementForIndexesNote' }),
         },
         columnWidths: { 'Holdings statement for indexes': '16%' },
         formatter: {
-          'Holdings statement for indexes': x => get(x, ['statement'], ''),
-          'Holdings statement for indexes note': x => get(x, ['note'], ''),
+          'Holdings statement for indexes': x => get(x, ['statement']) || noValue,
+          'Holdings statement for indexes note': x => get(x, ['note']) || noValue,
         },
         ariaLabel: intl.formatMessage({ id: 'ui-inventory.holdingsStatementForIndexes' }),
       },
     ];
 
+    const layoutNotes = content => {
+      const notesList = isEmpty(content) ? emptyList : content;
+      const orderedNotes = orderBy(notesList, ['noteType.name'], ['asc']);
+
+      return orderedNotes.map(({ noteType, notes }, i) => {
+        const noteName = noteType ? noteType.name : <FormattedMessage id="ui-inventory.unknownNoteType" />;
+
+        return (
+          <Row key={i}>
+            <MultiColumnList
+              key={i}
+              id={`list-holdings-notes-${i}`}
+              contentData={notesList === emptyList ? emptyList : notes}
+              visibleColumns={['Staff only', 'Note']}
+              columnMapping={{
+                'Staff only': <FormattedMessage id="ui-inventory.staffOnly" />,
+                'Note': notesList === emptyList ? <FormattedMessage id="ui-inventory.note" /> : noteName,
+              }}
+              columnWidths={{
+                'Staff only': '25%',
+                'Note': '75%',
+              }}
+              formatter={
+                {
+                  'Staff only': x => (notesList === emptyList ? noValue : formatterStaffOnly(x)),
```

Comment:
Do you need parentheses here?
```suggestion
                  'Staff only': x => notesList === emptyList ? noValue : formatterStaffOnly(x),
```

Reply:
LintError, in function which returns ternary condition should be in brackets